int16 id                 # track id
string label            # object class
float32 confidence      # confidence of result

float32 x               # normalized value of box top-left point coordinate x
float32 y               # normalized value of box top-left point coordinate y
float32 height          # normalized value of box height
float32 width           # normalized value of box width

float32 vel_x              # normalized velocity of box top-left point coordinate x
float32 vel_y              # normalized velocity of box top-left point coordinate y
float32 vel_height          # normalized velocity of box height
float32 vel_width           # normalized velocity of box width